src/Compute/Compute/Strategies/ComputeRp/SubResourceStrategy.cs

```diff
@@ -28,6 +28,10 @@ static class SubResourceStrategy
             this IEngine engine, ResourceConfig<AvailabilitySet> availabilitySet)
             => engine.GetSubResourceReference(availabilitySet);
 
+        public static SubResource GetReference(
```

Comment:
Just checking - will this allow us to specify either a Name (indicating it is in the same resource group) or a ResourceId for the ProximityPlacementGroup?

Reply:
No, it only accept names.